aa-software2112/SOEN390_SimpleCamera
# Feat/qr scanner backend #107

To test this, perform the following;

1. Navigate to www.the-qrcode-generator.com
2. Click "URL" at the top and type in www.facebook.com (or any other URL)
3. Run the app, hold steady, focus the camera on the Qr code, hold the phone some half-meter away from the QR code, hold the "swipe area" for 1 second until you see the orange spinner. At this point the picture should have been taken, you can lift your finger. Wait for the spinner to cease. When it does, there should be either a result allowing you to navigate to the QR code URL, or it will not work and you may try again.
